### DISCUSSION TITLE
Family Management

### DISCUSSION THREAD

**Discussion by einfall** [isFakeAccount/psnawp#93] [Ideas]
Hello,
is it possibile to add Family Management like users, time, extended time.

This is the url for familiy users:
https://accounts.api.playstation.com/api/family/v3/users/me/family/members

**Comment by einfall** [comment]
This url is for the current play time.

https://accounts.api.playstation.com/api/activityTracking/v3/users/"accountId"/usage/tracking

**Reply by isFakeAccount** [reply]
This complains about 

```json
{
   "error":{
      "code":1048842,
      "message":"access token does not contain the required scope(s)",
      "referenceId":"ece1b1fb-531a-46be-a68b-776af2e312b8"
   }
}
```
No idea about the scope. Where did you get this endpoint from? @einfall

**Comment by isFakeAccount** [comment]
I don't think it will be possible. On phone if you open family management it just opens it in browser. Also, it asks for password for confirmation. Both these things combined will make it almost impossible to implement the endpoint.

If you have some time see [CONTRIBUTING](https://github.com/isFakeAccount/psnawp/blob/master/CONTRIBUTING.md). Maybe you figure a way to get the endpoint for family management.
